oci/tests/integration/suite_test.go

```diff
@@ -82,8 +82,11 @@ var (
 	// kubernetes client of the created cluster.
 	testEnv *tftestenv.Environment
 
+	// deleteTag is the tag that will be deleted when testing the delete func
+	deleteTag = "test-delete"
+
 	// testImageTags are the tags used in the test for the generated images.
-	testImageTags = []string{"v0.1.0", "v0.1.2", "v0.1.3", "v0.1.4"}
+	testImageTags = []string{"v0.1.0", "v0.1.2", "v0.1.3", "v0.1.4", deleteTag}
```

Comment:
Since the other existing tests are very primitive, without any knowledge of scanned tags, we can just pick and delete any of the existing tags instead of introducing a new one. `deleteTag` can be set to any of the existing tags above.
Maybe in the future as we find a need for each test to have its own set of tags, we can move the image create and push operation into the test functions.